apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/server/RandomFissClaimGeneratorTest.java

```diff
@@ -24,321 +24,140 @@ public void randomClaim() throws InvalidProtocolBufferException {
-            + "  \"admDateCymd\": \"2021-02-22\",\n"
-            + "  \"admSourceUnrecognized\": \"p\",\n"
-            + "  \"primaryPayerCodeEnum\": \"PAYERS_CODE_VETERANS_ADMINISTRATION\",\n"
-            + "  \"attendPhysId\": \"hc5ft3r\",\n"
-            + "  \"attendPhysLname\": \"49cd\",\n"
-            + "  \"attendPhysFname\": \"ftsdx4\",\n"
-            + "  \"attendPhysMint\": \"t\",\n"
-            + "  \"attendPhysFlagUnrecognized\": \"v\",\n"
-            + "  \"operatingPhysId\": \"zxw89\",\n"
-            + "  \"operPhysLname\": \"5b\",\n"
-            + "  \"operPhysFname\": \"1zq84dp\",\n"
-            + "  \"operPhysMint\": \"k\",\n"
-            + "  \"operPhysFlagEnum\": \"PHYSICIAN_FLAG_NO\",\n"
-            + "  \"othPhysId\": \"x79m54mr\",\n"
-            + "  \"othPhysLname\": \"sj6k56gg\",\n"
-            + "  \"othPhysFname\": \"84r\",\n"
-            + "  \"othPhysMint\": \"6\",\n"
+            + "  \"originalXrefDcn\": \"p5fq7s8bcsxfrcg\",\n"
+            + "  \"paidDtCymd\": \"2021-01-09\",\n"
+            + "  \"admDateCymd\": \"2021-04-29\",\n"
+            + "  \"admSourceUnrecognized\": \"0\",\n"
+            + "  \"primaryPayerCodeEnum\": \"PAYERS_CODE_BLACK_LUNG\",\n"
+            + "  \"attendPhysId\": \"9fgh4d\",\n"
+            + "  \"attendPhysLname\": \"tf7h188107gzznnx9\",\n"
+            + "  \"attendPhysFname\": \"5vws5h\",\n"
+            + "  \"attendPhysMint\": \"9\",\n"
+            + "  \"attendPhysFlagUnrecognized\": \"c\",\n"
+            + "  \"operatingPhysId\": \"wwgghm8qjf2ns1p4\",\n"
+            + "  \"operPhysLname\": \"9hc5ft3rk49\",\n"
+            + "  \"operPhysFname\": \"dhftsdx4kt02\",\n"
+            + "  \"operPhysMint\": \"1\",\n"
+            + "  \"operPhysFlagUnrecognized\": \"w\",\n"
+            + "  \"othPhysId\": \"9k5b3\",\n"
+            + "  \"othPhysLname\": \"zq84dp\",\n"
+            + "  \"othPhysFname\": \"kvxb\",\n"
+            + "  \"othPhysMint\": \"7\",\n"
             + "  \"othPhysFlagEnum\": \"PHYSICIAN_FLAG_NO\",\n"
-            + "  \"xrefHicNbr\": \"hn70st6z5\",\n"
-            + "  \"procNewHicIndUnrecognized\": \"j\",\n"
-            + "  \"newHic\": \"73j4\",\n"
+            + "  \"xrefHicNbr\": \"4\",\n"
+            + "  \"procNewHicIndUnrecognized\": \"7\",\n"
+            + "  \"newHic\": \"j6k56gg1\",\n"
             + "  \"reposIndEnum\": \"REPOSITORY_INDICATOR_HIC_HAS_BEEN_MOVED\",\n"
-            + "  \"reposHic\": \"mb\",\n"
-            + "  \"mbiSubmBeneIndEnum\": \"FISS_HIC_OR_MBI_IS_MBI\",\n"
-            + "  \"adjMbiIndUnrecognized\": \"9\",\n"
-            + "  \"adjMbi\": \"19kn4mf5\",\n"
-            + "  \"medicalRecordNo\": \"kk\"\n"
+            + "  \"reposHic\": \"7\",\n"
+            + "  \"mbiSubmBeneIndUnrecognized\": \"z\",\n"
+            + "  \"adjMbiIndUnrecognized\": \"n\",\n"
+            + "  \"adjMbi\": \"0st6z50nj07\",\n"
+            + "  \"medicalRecordNo\": \"j\"\n"
```

Comment:
not expecting a change for this PR but should these be file resources instead? this seems really fiddly to me and a true diff of the response might be more useful during the reviews

Reply:
The problem is that a random claim generator is highly sensitive to changes in field order, adding new fields, removing fields, etc.  In a future PR I'd like to change these tests so they don't depend on the actual field values but just test that they are populated with proper lengths, etc.